exercises/concept/chessboard/.docs/introduction.md

```diff
@@ -0,0 +1,94 @@
+# Introduction
```

Comment:
I really like this introduction! You've explained it well.

Reply:
Thanks. The first part is a fork from the Elixir/Chessboard 👍